docs/data/base/components/checkbox/UnstyledCheckboxIndeterminate.js

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { styled } from '@mui/system';
+import { Checkbox as BaseCheckbox } from '@mui/base/Checkbox';
+import HorizontalRule from '@mui/icons-material/HorizontalRule';
+
+export default function UnstyledCheckboxIndeterminate() {
+  return (
+    <div style={{ display: 'flex', gap: 15 }}>
+      <Checkbox aria-label="Indeterminate checkbox" indeterminate>
+        <Indicator>
+          <HorizontalRuleIcon />
+        </Indicator>
+      </Checkbox>
+      <Checkbox aria-label="Indeterminate disabled checkbox" indeterminate disabled>
+        <Indicator>
+          <HorizontalRuleIcon />
+        </Indicator>
+      </Checkbox>
+    </div>
+  );
+}
+
+const blue = {
```

Comment:
Quick nit: should we remove all of the colors we're not using in these demos? There might be room to consolidate their usage as well (e.g., we're using blue 400, 500, 600, and 800) as the Checkbox is such a visually simple component 😅

Reply:
> should we remove all of the colors we're not using in these demos?

I'm down for getting rid of the unused colors. When I revamped the demos before, I left all the colors in there to make tweaks easier. But now, since our color palettes are more standardized, I think we should just stick to the ones we're actually using.